### PR TITLE
Converted view items to camel Case

### LINKED ISSUE #1330: Change the view items to camel case
**Actual Behaviour**

Currently all the view items that are referenced from the xml file do not follow the camel case

**Expected Behaviour**

the view items should follow the camel case implementation

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fixes #1330 

Changes: Added all the view items as camelCase

### CODE REVIEW

**Comment by iamareebjamal** [comment]
This is a nice opportunity to update the variable names to better declarative ones

File: app/src/main/res/layout/fragment_skill_listing.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
             android:layout_gravity="center" />
 
         <TextView
-            android:id="@+id/error_skill_fetch"
+            android:id="@+id/errorSkillFetch"
```
Comment (arundhati24) [inline app/src/main/res/layout/fragment_skill_listing.xml]: Snake case should be preferred for naming IDs.

File: app/src/main/res/layout/fragment_skill_details.xml [hunk 2]
```diff
@@ -241,14 +241,14 @@
...
-                android:id="@+id/skill_detail_policy"
+                android:id="@+id/skillDetailPolicy"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="16dp"
                 android:text="@string/developer_privacy_policy" />
 
             <TextView
-                android:id="@+id/skill_detail_terms"
+                android:id="@+id/skillDetailTerms"
```
Comment (arundhati24) [inline app/src/main/res/layout/fragment_skill_details.xml]: Snake case should be preferred for naming IDs in XML files.

File: app/src/main/res/layout-land/activity_login.xml [hunk 3]
```diff
@@ -42,7 +42,7 @@
                 app:errorEnabled="true">
 
                 <AutoCompleteTextView
-                    android:id="@+id/email_input"
+                    android:id="@+id/emailInput"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
```
Comment (arundhati24) [inline app/src/main/res/layout-land/activity_login.xml]: Snake case should be used as a convention for naming IDs in XML files.

File: app/src/main/res/layout-land/activity_login.xml [hunk 4]
```diff
@@ -57,7 +57,7 @@
                 app:passwordToggleEnabled="true">
 
                 <android.support.design.widget.TextInputEditText
-                    android:id="@+id/password_input"
+                    android:id="@+id/passwordInput"
                     android:layout_width="match_parent"
```
Comment (arundhati24) [inline app/src/main/res/layout-land/activity_login.xml]: Snake case

File: app/src/main/res/layout-land/activity_login.xml [hunk 5]
```diff
@@ -66,14 +66,14 @@
             </android.support.design.widget.TextInputLayout>
 
             <android.support.v7.widget.AppCompatCheckBox
-                android:id="@+id/custom_server"
+                android:id="@+id/customServer"
```
Comment (arundhati24) [inline app/src/main/res/layout-land/activity_login.xml]: Snake case

File: app/src/main/res/layout-land/activity_login.xml [hunk 6]
```diff
@@ -66,14 +66,14 @@
...
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginBottom="5dp"
                 android:text="@string/custom_server" />
 
             <android.support.design.widget.TextInputLayout
-                android:id="@+id/input_url"
+                android:id="@+id/inputUrl"
```
Comment (arundhati24) [inline app/src/main/res/layout-land/activity_login.xml]: Snake case

File: app/src/main/res/layout-land/activity_login.xml [hunk 7]
```diff
@@ -87,15 +87,15 @@
             </android.support.design.widget.TextInputLayout>
 
             <android.support.v7.widget.AppCompatButton
-                android:id="@+id/log_in"
+                android:id="@+id/logIn"
```
Comment (arundhati24) [inline app/src/main/res/layout-land/activity_login.xml]: Snake case

File: app/src/main/res/layout-land/activity_login.xml [hunk 8]
```diff
@@ -87,15 +87,15 @@
...
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
                 android:background="@drawable/rounded_button"
                 android:text="@string/activity_login_log_in" />
 
             <TextView
-                android:id="@+id/forgot_password"
+                android:id="@+id/forgotPassword"
```
Comment (arundhati24) [inline app/src/main/res/layout-land/activity_login.xml]: Snake case

File: app/src/main/res/layout/fragment_skill_listing.xml (hunk 1 above)
Comment (iamareebjamal) [reply app/src/main/res/layout/fragment_skill_listing.xml]: No, it shouldn't when using those variable names in Kotlin

**Comment by iamareebjamal** [comment]
@batbrain7 Please change names to better reflect their nature. Otherwise, LGTM

File: app/src/main/res/layout/fragment_skill_listing.xml (hunk 1 above)
Comment (arundhati24) [reply app/src/main/res/layout/fragment_skill_listing.xml]: @iamareebjamal 👍 Thanks for the clarification.

**Comment by iamareebjamal** [comment]
@batbrain7 Status?

**Comment by batbrain7** [comment]
@iamareebjamal did the changes

File: app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt [hunk 9]
```diff
@@ -306,40 +306,40 @@ class SkillDetailsFragment : Fragment() {
...
-            skillDetailPolicy.linksClickable = true
-            skillDetailPolicy.movementMethod = LinkMovementMethod.getInstance()
+            tvSkillDetailPolicy.linksClickable = true
+            tvSkillDetailPolicy.movementMethod = LinkMovementMethod.getInstance()
             if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
-                skillDetailPolicy.text = Html.fromHtml("<a href=\"${skillData.developerPrivacyPolicy}\">Developer's Privacy Policy</a>", Html.FROM_HTML_MODE_COMPACT)
+                tvSkillDetailPolicy.text = Html.fromHtml("<a href=\"${skillData.developerPrivacyPolicy}\">Developer's Privacy Policy</a>", Html.FROM_HTML_MODE_COMPACT)
             } else {
-                skillDetailPolicy.text = Html.fromHtml("<a href=\"${skillData.developerPrivacyPolicy}\">Developer's Privacy Policy</a>")
+                tvSkillDetailPolicy.text = Html.fromHtml("<a href=\"${skillData.developerPrivacyPolicy}\">Developer's Privacy Policy</a>")
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt]: Remove tv and et from the fields

File: app/src/main/java/org/fossasia/susi/ai/signup/SignUpActivity.kt [hunk 10]
```diff
@@ -137,7 +137,7 @@ class SignUpActivity : AppCompatActivity(), ISignUpView {
     }
 
     override fun enableSignUp(bool: Boolean) {
-        signUp?.isEnabled = bool
+        btnSignUp?.isEnabled = bool
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/signup/SignUpActivity.kt]: It was better earlier

**Comment by batbrain7** [comment]
@iamareebjamal made the requested changes

**Pull request merged**